layers/+tags/gtags/packages.el

```diff
@@ -11,13 +12,24 @@
 
 (setq gtags-packages
   '(
+    eldoc
     helm-gtags
     ggtags
     ))
 
 (defun gtags/init-ggtags ()
   (use-package ggtags
-    :defer t))
+    :defer t
+    :init
+    (progn
+      ;; modes that do not have a layer, add here.
+      (add-hook 'awk-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'shell-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'tcl-mode-hook (lambda () (ggtags-mode 1)))
+      (add-hook 'vhdl-mode-hook (lambda () (ggtags-mode 1)))
```

Comment:
Joe Hillenbrand <notifications@github.com> writes:
>> +      (add-hook 'vhdl-mode-hook (lambda () (ggtags-mode 1)))
>
> I believe it's preferred to use a named function when adding hooks rather than a lambda for debugging purposes.

I've heard this about hooks as well; however, I don't believe this is a
spacemacs design requirement, and I don't think its meant to apply to
the simple case of a single function call.

Thanks,
Chris.